### PR TITLE
Activity masonry

### PR DESCRIPTION
**Pull request by raftaar1191**

### CODE REVIEW

File: app/admin/RTMediaFormHandler.php [hunk 1]
```diff
@@ -355,6 +355,21 @@ static function display_render_options( $options ) {
 			),
 		);
 
+		if( function_exists( 'bp_is_active' ) ){
```
Comment (chandrapatel) [inline app/admin/RTMediaFormHandler.php]: Add spacing. like this `if ( function_exists( 'bp_is_active' ) ) {`

File: app/admin/RTMediaFormHandler.php [hunk 2]
```diff
@@ -355,6 +355,21 @@ static function display_render_options( $options ) {
...
+			$render[ 'rtmedia-enable-masonry-activity' ] = array(
```
Comment (chandrapatel) [inline app/admin/RTMediaFormHandler.php]: remove space before/after `[]` like this `$render['rtmedia-enable-masonry-activity']`

File: app/admin/RTMediaFormHandler.php [hunk 3]
```diff
@@ -355,6 +355,21 @@ static function display_render_options( $options ) {
...
+				'title' => esc_html__( 'Enable Masonry in Activity & Group', 'buddypress-media' ),
+				'callback' => array( 'RTMediaFormHandler', 'checkbox' ),
+				'args' => array(
+					'key' => 'buddypress_enableMasonryActivity',
+					'value' => $options[ 'buddypress_enableMasonryActivity' ],
```
Comment (chandrapatel) [inline app/admin/RTMediaFormHandler.php]: remove space before/after `[]`. Like `$options['buddypress_enableMasonryActivity']`

File: app/admin/RTMediaFormHandler.php (hunk 1 above)
Comment (BhargavBhandari90) [inline app/admin/RTMediaFormHandler.php]: Done

**Comment by BhargavBhandari90** [comment]
New PR for this is : https://github.com/rtMediaWP/rtMedia/pull/858

File: app/admin/RTMediaFormHandler.php (hunk 3 above)
Comment (BhargavBhandari90) [inline app/admin/RTMediaFormHandler.php]: Done

File: app/admin/RTMediaFormHandler.php (hunk 2 above)
Comment (BhargavBhandari90) [inline app/admin/RTMediaFormHandler.php]: Done

**Comment by BhargavBhandari90** [comment]
New PR for this is https://github.com/rtMediaWP/rtMedia/pull/887
So closing this.

**Pull request closed without merging**